doc/changes/latest.inc

```diff
@@ -69,6 +69,8 @@ Bugs
 
 - Fix bug in :func:`mne.viz.plot_alignment` where ECoG and sEEG channels were not plotted and fNIRS channels were always plotted in the head coordinate frame by `Eric Larson`_ (:gh:`8393`)
 
+- Fix :func:`mne.io.read_raw_curry` to deal with Curry datasets that have channels that are listed in the labels file, but which are absent from the saved data file (e.g. 'Ref' channel).  Also now populates info['meas_date'] if possible by `Tod Flak` _ (:gh:`8400`)
```

Comment:
For this link to work you'll need to add your name to 

https://github.com/mne-tools/mne-python/blob/master/doc/changes/names.inc

And because you're a new contributor (congrats!) you should put the entry at the top of the section and format like we did for the change by `Eduard Ort` above